config/environments/cardona/node.config.toml

```diff
@@ -0,0 +1,92 @@
+[Log]
+Environment = "development" # "production" or "development"
+Level = "info"
+Outputs = ["stderr"]
+
+[State]
+AccountQueue = 64
```

Comment:
This parameter must be removed from the [State] section. We need also to add this 2 config params in the Pool section:
[Pool]
AccountQueue = 64
GlobalQueue = 1024

This was fixed by me some time ago for the debug/test toml files and in default.go but it seems I didn't reviewed the environments config files. This change must be also done for testnet and mainnet config files, we can do this change in this PR